src/qt/window.cpp

```diff
@@ -1582,6 +1585,39 @@ QWidget *wxWindowQt::GetHandle() const
     return m_qtWindow;
 }
 
+void wxWindowQt::SetMinSize(const wxSize& minSize)
+{
+    wxWindowBase::SetMinSize(minSize);
+    const int width = minSize.GetWidth();
+    const int height = minSize.GetHeight();
+    QWidget* widget = GetHandle();
+    if ( widget != NULL )
+       widget->setMinimumSize(QSize(width == -1 ? 0 : width, height == -1 ? 0 : height));
```

Comment:
I could be missing something, but I don't think it's the right thing to do. [Qt docs](https://doc.qt.io/qt-5/qwidget.html#minimumSize-prop) indicate the the minimum size prevents the window from becoming smaller than this size even if `SetSize()` is explicitly called on this, but this is not how it works in wx API: min/max size only restrict the size that the user can resize the window to, the program can always set whatever size it wants.

So I just don;t think we should do this at all. What's the rationale for this change, exactly?